en/faq/cached-components.md

```diff
@@ -1,26 +1,27 @@
 ---
-title: Caching Components
-description: How to cache components?
+title: Composants avec cache
+description: Comment mettre en cache des composants ?
 ---
 
-# How to cache Vue components?
+# Comment mettre en cache des composants Vue ?
 
-> Although Vue's SSR is quite fast, it can't match the performance of pure string-based templating due to the cost of creating component instances and Virtual DOM nodes. In cases where SSR performance is critical, wisely leveraging caching strategies can greatly improve response time and reduce server load.
+> Bien que le rendu côté serveur de Vue soit rapide, il ne rivalise pas avec les performances d'un template basé sur une chaine de caractère pure, et ceux, à cause du coût de création des instances de composant et des nœuds du DOM virtuel. Dans le cas où les performances du rendu côté serveur est critique, mettre en place une bonne stratégie de mise en cache peut grandement améliorer le temps de réponse et réduire la charge serveur.
 
-To avoid boilerplate, use [Component Cache module](https://github.com/nuxt-community/modules/tree/master/packages/component-cache) for Nuxt.js.
-This module uses vue-server-renderer to add LRU cache support for Vue components.
+Vous pouvez utiliser le module [Component Cache](https://github.com/nuxt-community/modules/tree/master/packages/component-cache) pour Nuxt.js.
+Ce module utilise `vue-server-renderer` pour ajouter le support d'un cache [LRU](https://fr.wikipedia.org/wiki/Algorithmes_de_remplacement_des_lignes_de_cache#LRU_.28Least_Recently_Used.29) pour les composants Vue.
 
-## Usage
+## Utilisation
+
+- Ajoutez la dépendance `@nuxtjs/component-cache` en utilisant Yarn ou npm pour votre projet.
+- Ajoutez `@nuxtjs/component-cache` to `modules` section of `nuxt.config.js`.
```

Comment:
Ajoutez `@nuxtjs/component-cache` ~~to~~ **à la section** `modules` ~~section of~~ **de** `nuxt.config.js`.